internal/webhookcert/webhook_cert_test.go

```diff
@@ -127,13 +133,49 @@ var (
 			},
 		},
 	}
+
+	mutatingWebhookConfiguration = admissionregistrationv1.MutatingWebhookConfiguration{
```

Comment:
can we have both variables named `initialValidatingWebhookConfiguration` and `initialMutatingWebhookConfiguration` or both of them without the `initial` prefix?

Reply:
I removed prefix